docs/index.txt

```diff
@@ -9,62 +9,172 @@ Laravel MongoDB
+   --------------
+
+   For runnable code examples and explanations of common MongoDB operations,
+   see :ref:`laravel-usage-examples`.
+
+Fundamentals
+------------
+
+To learn how to perform the following tasks by using the {+odm-short+},
+see the following content:
 
-- :ref:`laravel-install`
 - :ref:`laravel-eloquent-models`
 - :ref:`laravel-query-builder`
 - :ref:`laravel-user-authentication`
 - :ref:`laravel-queues`
 - :ref:`laravel-transactions`
-- :ref:`laravel-upgrading`
+
+Upgrade Versions
+----------------
+
+Learn what changes you might need to make to your application to upgrade
+versions in the :ref:`laravel-upgrading` section.
+
+..
+
+   FAQ
+   ---
+
+   To learn about solutions to commonly asked questions about the {+odm-short+},
+   see the :ref:`laravel-faq` section.
+
+   Connection Troubleshooting
+   --------------------------
+
+   To learn how to solve issues you might have when connecting to MongoDB, see
+   :ref:`laravel-connection-troubleshooting`.
+
+   Error Handling
+   --------------
+
+   To learn about errors you might encounter when you use {+odm-short+}
+   to perform MongoDB operations, see :ref:`laravel-error-handling`.
+
+   Issues & Help
+   -------------
+
+   Learn how to report bugs, contribute to {+odm-short+}, and find
+   additional resources for asking questions in the
+   :ref:`laravel-issues-and-help` section.
+
+   Compatibility
+   -------------
+
+   To learn about compatibility between {+odm-short+} and the following
+   components, see :ref:`laravel-compatibility`:
+
+   - MongoDB (TBD)
+   - Laravel
+   - Laravel Eloquent features
+
+   TODO: Move the content in "Reporting Issues", "Security Vulnerabilities",
+   and "Development" to "Issues & Help".
 
 Reporting Issues
 ----------------
 
-Think you’ve found a bug in the library? Want to see a new feature? Please open a case in our issue management tool, JIRA:
+We are lucky to have a vibrant PHP community that includes users of varying
+experience with MongoDB PHP Library and {+odm-short+}. To get support for
+general questions, search or post in the
+`MongoDB Community Forums <https://www.mongodb.com/community/forums/>`__.
 
-- `Create an account and login <https://jira.mongodb.org/>`__
-- Navigate to the `PHPORM <https://jira.mongodb.org/browse/PHPORM>`__ project.
-- Click Create
-- Please provide as much information as possible about the issue type and how to reproduce it.
+To learn more about MongoDB support options, see the
+`Technical Support <https://www.mongodb.org/about/support>`__ page.
 
-Note: All reported issues in JIRA project are public.
 
-For general questions and support requests, please use one of MongoDB's 
-:manual:`Technical Support </support/>` channels.
+Bugs / Feature Requests
+-----------------------
 
-Security Vulnerabilities
-~~~~~~~~~~~~~~~~~~~~~~~~
+If you think you've found a bug or want to see a new feature in {+odm-short+},
+please open a case in our issue management tool, JIRA, by performing the
```

Comment:
@alcaeus I think we can tell people to create GitHub issues instead of Jira tickets. It's easier to enable external contributions.

Reply:
Sounds good to me. Do we need confirmation from @alcaeus?

Reply:
Yes, I'd like to keep GitHub issues open as long as we can. We have the same process across all of the PHP repos to reduce friction, and as long as the issue quality stays high I don't mind people creating issues directly in GitHub.